docs/contributing/contributing-documentation.md

```diff
@@ -0,0 +1,137 @@
+# Contributing to documentation
+
+We welcome contributions to Netdata's already extensive documentation, which we host at [docs.netdata.cloud](https://docs.netdata.cloud/) and store inside of the [main repository](https://github.com/netdata/netdata) on GitHub.
+
+Like all contributing to all other aspects of Netdata, we ask that anyone who wants to help with documentation read and abide by the [Contributor Convenant Code of Conduct](https://docs.netdata.cloud/code_of_conduct/) and follow the instructions outlined in our [Contributing document](../../CONTRIBUTING.md).
+
+We also ask you to read our [documentation style guide](style-guide.md), which, while not complete, will give you some guidance on how we write and organize our documentation.
+
+All our documentation uses the Markdown syntax. If you're not familiar with how it works, please read the [Markdown introduction post](https://daringfireball.net/projects/markdown/) by its creator, followed by [Mastering Markdown](https://guides.github.com/features/mastering-markdown/) guide from GitHub.
+
+
+## How contributing to the documentation works
+
+There are two ways to contribute to the Netdata contribution. 
```

Comment:
When I read this sentence, I confess I did not like, because it looks like a great pleonasm , but I decided to not say nothing, because I am not specialist in English, but you used at the end of the text the word " metaphor" and I changed my mind, because I saw we have more or less the same structure among the languages. My suggestion is do not use "to contribute" and "contribution" so close, I do not think there is necessity to use the last word("contribution") here.